app/src/main/java/com/sun_asterisk/moviedb_50/data/source/remote/Repository.kt

```diff
@@ -0,0 +1,33 @@
+package com.sun_asterisk.moviedb_50.data.source.remote
+
+import com.sun_asterisk.moviedb_50.data.model.Genres
+import com.sun_asterisk.moviedb_50.data.model.Movie
+import com.sun_asterisk.moviedb_50.data.source.DataSource
+
+
+class Repository private constructor(
+    private val mRemoteDataSource: DataSource.RemoteDataSource?,
```

Comment:
```suggestion
    private val remoteDataSource: DataSource.RemoteDataSource?,
```
convention kotlin k cần "m" ở prefix đâu

Reply:
mình nghĩ là DataSource.RemoteDataSource cần để là Not-Nullable
```suggestion
    private val remoteDataSource: DataSource.RemoteDataSource,
```